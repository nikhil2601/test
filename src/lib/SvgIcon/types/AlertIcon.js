import React from 'react';
import PropTypes from 'prop-types';

import SvgIcon from '../SvgIcon';

const AlertIcon = ({ setRef, ...rest }) => (
    <SvgIcon {...rest} ref={setRef}>
        <path d="M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z" />
    </SvgIcon>
);

AlertIcon.propTypes = {
    setRef: PropTypes.object,
    viewBox: PropTypes.string,
};

AlertIcon.defaultProps = {
    setRef: null,
    viewBox: '0 0 24 24',
};

export default AlertIcon;
