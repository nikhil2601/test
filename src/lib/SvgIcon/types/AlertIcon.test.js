import React from 'react';
import { shallow } from 'enzyme';

import AlertIcon from './AlertIcon';

describe('<AlertIcon />', () => {
    const wrapper = shallow(<AlertIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
