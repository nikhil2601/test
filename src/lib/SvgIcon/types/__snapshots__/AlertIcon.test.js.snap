// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 24 24"
  width="1rem"
>
  <path
    d="M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z"
  />
</SvgIcon>
`;
